providers/base/bin/camera_test.py

```diff
@@ -711,14 +955,17 @@ def get_video_devices():
         gi.require_version("GLib", "2.0")
         from gi.repository import GLib
 
-        gi.require_version("Clutter", "1.0")
-        from gi.repository import Clutter
+        Gst.init(None)
+        camera.Gst = Gst
+        camera.GLib = GLib
```

Comment:
I don't understand why this is not done in the function itself (calling a subfunction like `init_glib` or something). Else here there is a object attribute that pops into existence here for *some* tests and it is not clear why/when.

Reply:
I like this approach better. No good reason for implementing it that way. We were importing it globally before and I just added these libraries as object attributes for ease of testing. I will create the init_glib function